store/tikv/mock-tikv/mvcc.go

```diff
@@ -141,6 +141,25 @@ func (e *mvccEntry) Commit(startTS, commitTS uint64) error {
 	return nil
 }
 
+func (e *mvccEntry) FastCommit(mutation *kvrpcpb.Mutation, startTS, commitTS uint64) error {
+	if len(e.values) > 0 {
+		if e.values[0].commitTS >= startTS {
+			return ErrRetryable("write conflict")
+		}
+	}
+	if e.lock != nil {
+		return e.lockErr()
+	}
+	if mutation.GetOp() != kvrpcpb.Op_Lock {
```

Comment:
I think we need a TODO here. It may not satisfy SelectForUpdate in some scenarios.